tests/benchmarks/deterministic/BF/elevators_ppltl/elevators_teg.json

```diff
@@ -1,31 +1,31 @@
+    "s2-0": "O(\"served p1\") & O(\"served p0\" & !(Y(O(\"served p1\"))))",
```

Comment:
now formulas look like this...

in JSON we cannot use `'` to wrap strings; therefore, escaping `"` is the only way to specify the temporal goal.